docs/gl_objects/users.rst

```diff
@@ -84,6 +89,15 @@ Delete an external identity by provider name::
 
     user.identityproviders.delete('oauth2_generic')
 
+Get the followers of a user
+
+    user.followers_users.list()
+
+Get the followings of a user
+
+    user.following_users.list()
```

Comment:
@nejch I couldn't use `following` and `followers` directly, as it has the same name as it's user attribute.

Reply:
What do you think about this naming?

Reply:
Then let's keep the clearer name I guess 😄